tools/edid/generate-rpi-edid-dumps

```diff
@@ -5,51 +5,82 @@
 # Copyright (C) 2017 Kano Computing Ltd.
 # License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
 #
-# Generate three files: raw EDID, tvservice & edidparser dumps, and expected
+# Use raw EDIDs to generate two files: tvservice & edidparser dumps, and expected
 # kano_settings.system.display function outputs. These files are then added to
 # res/edid in the project and part of our testing dataset.
 #
-# This script is intended to be run on a RPi without sudo.
+# Each raw EDID is flashed in turn to the attached display in order for the system
+# as a whole to think it's using the respective display. The original screen EDID will
+# be backed up and restored at the end (see ORIGINAL_EDID_BACKUP_PATH for the path).
+#
+# This tool requires the installation of edid-rw under /usr/bin.
+# https://github.com/bulletmark/edid-rw
+#
+# WARNING: This tool can destroy the machine and display!
+#          See edid-rw README for the dangers of using it.
+#
```

Comment:
Would it be better to move these important considerations into the docopt help section?

Reply:
Good point! Done